src/main/scala/cromwell/engine/backend/CallMetadata.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.engine.backend
+
+import cromwell.binding.values.WdlFile
+import org.joda.time.DateTime
+
+case class CallMetadata(inputs: Map[String, String],
+                        start: DateTime,
+                        end: DateTime,
+                        jobid: String,
+                        rc: Int,
```

Comment:
We might not have an rc, so I'm thinking this should maybe be changed to a String which could be left empty in that case.